src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/SimpleTypeModelBinder.cs

```diff
@@ -67,7 +84,7 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
             {
                 bindingContext.ModelState.TryAddModelError(
                     bindingContext.ModelName,
-                    exception,
+                    exception.InnerException ?? exception,
```

Comment:
Why is this here? I know this was calling into another method before, what's the **important** detail? We should capture that here.